docs/DeveloperGuide.adoc

```diff
@@ -997,6 +997,155 @@ Use case ends.
 ** 2a1. Club Manager shows an error message.
 Use case resumes at step 1.
 
+[discrete]
+=== Use case: activityEdit
+
+*Main Success Scenario (MSS)*
+
+1.  User requests to list activities
+2.  Club Manager shows a list of activities
+3.  User requests to edit details of an activity in the displayed list
+4.  Club Manager edits the details of the specified member with given information.
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. The list is empty
++
+Use case ends.
+
+* 3a. The given index or parameters is/are invalid.
++
+** 3a1. Club Manager shows an error message.
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: activitySort
+
+*Main Success Scenario (MSS)*
+
+1.  Admin requests to sort the activity list
+2.  Club Manager sorts the activity list according to specified parameter
+3.  Club Management System shows the sorted list.
```

Comment:
typo here?

Reply:
oh yes, sorry i'll edit it and add the command summary as well then update on my repo.